stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -40,6 +40,9 @@ spec:
           args:
             - /nginx-ingress-controller
             - --default-backend-service={{ if .Values.defaultBackend.enabled }}{{ .Release.Namespace }}/{{ template "nginx-ingress.defaultBackend.fullname" . }}{{ else }}{{ .Values.controller.defaultBackendService }}{{ end }}
+          {{- if (eq "0.9.0" .Values.controller.image.tag) }}
```

Comment:
This will only work for 0.9.0. 0.9.1, for example, won't have the annotation prefix applied. Did you intent for it to be a range of versions? Version comparison functions are available... https://masterminds.github.io/sprig/semver.html